connectors/webhook.go

```diff
@@ -38,88 +38,103 @@ func (x Webhook) Listen(commandMsgs chan<- models.Message, connector models.Conn
 		log.Printf("Starting Webhook connector at: %s", server.Addr)
 	}
 
-	http.HandleFunc("/webhook/", webhookHandler)
+	http.HandleFunc("/", webhookHandler)
```

Comment:
We could split this into 
http.HandleFunc("/command", commandHandler) 
http.HandleFunc("/publish", publishHandler)